src/asciidocParser.ts

```diff
@@ -125,43 +134,7 @@ export class AsciidocParser {
       highlightjsBuiltInSyntaxHighlighter.$register_for('highlight.js', 'highlightjs')
     }
 
-    let attributes = {}
-
-    if (containsStyle) {
-      attributes = {}
-    } else if (previewStyle !== '') {
-      let stylesdir: string, stylesheet: string
-
-      if (path.isAbsolute(previewStyle)) {
-        stylesdir = path.dirname(previewStyle)
-        stylesheet = path.basename(previewStyle)
-      } else {
-        if (workspacePath === undefined) {
-          stylesdir = ''
-        } else if (workspacePath.length > 0) {
-          stylesdir = workspacePath[0].uri.path
-        }
-
-        stylesdir = path.dirname(path.join(stylesdir, previewStyle))
-        stylesheet = path.basename(previewStyle)
-      }
-
-      attributes = {
-        stylesdir: stylesdir,
-        stylesheet: stylesheet,
-      }
-    } else if (useEditorStylesheet) {
-      attributes = {
-        'allow-uri-read': true,
-        stylesdir: this.stylesdir,
-        stylesheet: 'asciidoctor-editor.css',
-      }
-    } else {
-      attributes = {
-        stylesdir: this.stylesdir,
-        stylesheet: 'asciidoctor-default.css@',
-      }
-    }
+    const attributes = {}
 
     // TODO: Check -- Not clear that this code is functional
```

Comment:
The function works for me if I set in `settings.json`:

    "asciidoc.preview.attributes": {
        "test": "bla222h",
    }

Reply:
I will remove the comment then 👍🏻